dwitter/static/js/feed.js

```diff
@@ -1,4 +1,5 @@
 var onDweetChanged = function() {
+  // eslint-disable-next-line newline-per-chained-call
   var charCount = $(this).parent().parent().parent().find('.character-count')[0];
```

Comment:
The Airbnb ESLint config has a rule against chaining more than three method calls together. In this case, I don't think it hurts to chain four method calls.